aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -102,6 +102,12 @@ pub use aws_types::{
 /// Load default sources for all configuration with override support
 pub use loader::ConfigLoader;
 
+/// Types for configuring identity caching.
+pub mod identity {
+    pub use aws_smithy_runtime::client::identity::IdentityCache;
```

Comment:
I don't think it belongs in aws-smithy-runtime-api since that crate is intended for library authors, and concrete built-in identity cache implementations aren't needed for that. So leaves us with two options: 1. put them in aws-smithy-runtime, or 2. create a new crate for them.

I opted to go for 1, but that probably means we have to make aws-smithy-runtime a stable crate.